landmark_gnss:
    - [367941.045549, 3955739.19975]
    - [367941.045549, 3955743.19975]
goal_point:
    - [367949.045549, 3955737.19975]
    - [367935.403171, 3955734.10032]
    - [367935.925485, 3955742.10032]
    - [367951.925485, 3955742.10032]
    - [367951.403171, 3955734.10032]
#goal_point:
#    - [367941.840248, 3955736.04768]
#    - [367942.874767, 3955728.78041]
#    - [0.0, 0.0]
#    - [12.0, 0.0]
#    - [12.0, -5.0]
#    - [0.0, -5.0]
#    - [0.0, 0.0001]
obstacle_position:
    - [4.0, -2.0]
    - [8.0, -2.0]
    - [12.0, -2.0]
    - [4.0, -6.0]
    - [8.0, -6.0]
    - [12.0, -6.0]
#obstacle_position:
#    - [3.0, -1.0]
#    - [6.0, -1.0]
#    - [9.0, -1.0]
#    - [3.0, -4.0]
#    - [6.0, -4.0]
#    - [9.0, -4.0]
#367956.77323135035, 3955733.45885006
#367952.8518598575, 3955732.6696443735
#367948.1412826784, 3955731.8804386873
#367955.1948199777, 3955737.380221553
#367951.27344848483, 3955736.5910158665
#367947.35207699204, 3955735.8018101803
